swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
```

Comment:
👍 We needed to explicitly import `Item` here, since we list it by name as the type `Clothing` inherits from.